AMRProfParser compile error

When I try to build the AMRProfParser with GCC 11.3.0 (I have also tried GCC 13.1.1 and Clang 16.0.5), I get the following compiler error:
```
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp: In member function 'void amrex::DataServices::RunTimelinePF(std::map<int, std::__cxx11::basic_string<char> >&, std::string&, BLProf
Stats::TimeRange&, int, int, int, bool&)':
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:2491:32: error: no matching function for call to 'pow(int, int)'                                                                   
 2491 |             ntnMultiplier = pow(10, static_cast<int>( 1 + log10(ntnSize)));
      |                             ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../../../amrex/Src/Base/AMReX_ParallelDescriptor.H:13,
                 from ../../../amrex/Src/Base/AMReX_DistributionMapping.H:10,
                 from ../../../amrex/Src/Base/AMReX_Utility.H:13,
                 from ../../../amrex/Src/Base/AMReX_BaseFab.H:19,
                 from ../../../amrex/Src/Base/AMReX_FArrayBox.H:6,
                 from ../../../amrex/Src/Base/AMReX_MultiFab.H:7,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_AmrData.H:11,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.H:10,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:5:
../../../amrex/Src/Base/AMReX_GpuComplex.H:415:15: note: candidate: 'template<class T> amrex::GpuComplex<T> amrex::pow(const amrex::GpuComplex<T>&, const T&)'
  415 | GpuComplex<T> pow (const GpuComplex<T>& a_z, const T& a_y) noexcept
      |               ^~~
../../../amrex/Src/Base/AMReX_GpuComplex.H:415:15: note:   template argument deduction/substitution failed:
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:2491:32: note:   mismatched types 'const amrex::GpuComplex<T>' and 'int'
 2491 |             ntnMultiplier = pow(10, static_cast<int>( 1 + log10(ntnSize)));
      |                             ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../../../amrex/Src/Base/AMReX_ParallelDescriptor.H:13,
                 from ../../../amrex/Src/Base/AMReX_DistributionMapping.H:10,
                 from ../../../amrex/Src/Base/AMReX_Utility.H:13,
                 from ../../../amrex/Src/Base/AMReX_BaseFab.H:19,
                 from ../../../amrex/Src/Base/AMReX_FArrayBox.H:6,
                 from ../../../amrex/Src/Base/AMReX_MultiFab.H:7,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_AmrData.H:11,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.H:10,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:5:
../../../amrex/Src/Base/AMReX_GpuComplex.H:448:15: note: candidate: 'template<class T> amrex::GpuComplex<T> amrex::pow(const amrex::GpuComplex<T>&, int)'
  448 | GpuComplex<T> pow (const GpuComplex<T>& a_z, int a_n) noexcept
      |               ^~~
../../../amrex/Src/Base/AMReX_GpuComplex.H:448:15: note:   template argument deduction/substitution failed:
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:2491:32: note:   mismatched types 'const amrex::GpuComplex<T>' and 'int'
 2491 |             ntnMultiplier = pow(10, static_cast<int>( 1 + log10(ntnSize)));
      |                             ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:2504:31: error: no matching function for call to 'pow(int, int)'
 2504 |             bnMultiplier = pow(10, static_cast<int>( 1 + log10(bnSize)));
      |                            ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../../../amrex/Src/Base/AMReX_ParallelDescriptor.H:13,
                 from ../../../amrex/Src/Base/AMReX_DistributionMapping.H:10,
                 from ../../../amrex/Src/Base/AMReX_Utility.H:13,
                 from ../../../amrex/Src/Base/AMReX_BaseFab.H:19,
                 from ../../../amrex/Src/Base/AMReX_FArrayBox.H:6,
                 from ../../../amrex/Src/Base/AMReX_MultiFab.H:7,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_AmrData.H:11,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.H:10,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:5:
../../../amrex/Src/Base/AMReX_GpuComplex.H:415:15: note: candidate: 'template<class T> amrex::GpuComplex<T> amrex::pow(const amrex::GpuComplex<T>&, const T&)'
  415 | GpuComplex<T> pow (const GpuComplex<T>& a_z, const T& a_y) noexcept
      |               ^~~
../../../amrex/Src/Base/AMReX_GpuComplex.H:415:15: note:   template argument deduction/substitution failed:
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:2504:31: note:   mismatched types 'const amrex::GpuComplex<T>' and 'int'
 2504 |             bnMultiplier = pow(10, static_cast<int>( 1 + log10(bnSize)));
      |                            ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../../../amrex/Src/Base/AMReX_ParallelDescriptor.H:13,
                 from ../../../amrex/Src/Base/AMReX_DistributionMapping.H:10,
                 from ../../../amrex/Src/Base/AMReX_Utility.H:13,
                 from ../../../amrex/Src/Base/AMReX_BaseFab.H:19,
                 from ../../../amrex/Src/Base/AMReX_FArrayBox.H:6,
                 from ../../../amrex/Src/Base/AMReX_MultiFab.H:7,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_AmrData.H:11,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.H:10,
                 from ../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:5:
../../../amrex/Src/Base/AMReX_GpuComplex.H:448:15: note: candidate: 'template<class T> amrex::GpuComplex<T> amrex::pow(const amrex::GpuComplex<T>&, int)'
  448 | GpuComplex<T> pow (const GpuComplex<T>& a_z, int a_n) noexcept
      |               ^~~
../../../amrex/Src/Base/AMReX_GpuComplex.H:448:15: note:   template argument deduction/substitution failed:
../../../amrex/Src/Extern/amrdata/AMReX_DataServices.cpp:2504:31: note:   mismatched types 'const amrex::GpuComplex<T>' and 'int'
 2504 |             bnMultiplier = pow(10, static_cast<int>( 1 + log10(bnSize)));
      |                            ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
I think it just needs the `std::` namespace qualifier on the `pow()`s.

--- a/.gitignore
+++ b/.gitignore
@@ -62,3 +62,9 @@ Tutorials/EB/Donut/Exec/ls_plt*
 # local config
 Tools/GNUMake/Make.local
 Tools/GNUMake/Make.local-pre
+
+# profparser
+BLProfParser.lex.yy.cpp
+BLProfParser.tab.H
+BLProfParser.tab.cpp
+BLProfParserNC.y

--- a/Src/Extern/amrdata/AMReX_DataServices.cpp
+++ b/Src/Extern/amrdata/AMReX_DataServices.cpp
@@ -2488,7 +2488,7 @@ void DataServices::RunTimelinePF(std::map<int, std::string> &mpiFuncNames,
           int ntnSize(nameTagNames.size());
           ntnNumbers.resize(ntnSize, 0.0);
           if(ntnSize > 0) {
-            ntnMultiplier = std::pow(10, static_cast<int>( 1 + log10(ntnSize)));
+            ntnMultiplier = std::pow(10, static_cast<int>( 1 + std::log10(ntnSize)));
           }
           for(int i(0); i < ntnSize; ++i) {
             if(ntnMultiplier > 0.0) {
@@ -2501,7 +2501,7 @@ void DataServices::RunTimelinePF(std::map<int, std::string> &mpiFuncNames,
           int bnSize(barrierNames.size());
           bnNumbers.resize(bnSize, 0.0);
           if(bnSize > 0) {
-            bnMultiplier = std::pow(10, static_cast<int>( 1 + log10(bnSize)));
+            bnMultiplier = std::pow(10, static_cast<int>( 1 + std::log10(bnSize)));
           }
           for(int i(0); i < bnSize; ++i) {
             if(bnMultiplier > 0.0) {

--- a/Src/Extern/amrdata/AMReX_DataServices.cpp
+++ b/Src/Extern/amrdata/AMReX_DataServices.cpp
@@ -2488,7 +2488,7 @@ void DataServices::RunTimelinePF(std::map<int, std::string> &mpiFuncNames,
           int ntnSize(nameTagNames.size());
           ntnNumbers.resize(ntnSize, 0.0);
           if(ntnSize > 0) {
-            ntnMultiplier = pow(10, static_cast<int>( 1 + log10(ntnSize)));
+            ntnMultiplier = std::pow(10, static_cast<int>( 1 + log10(ntnSize)));
           }
           for(int i(0); i < ntnSize; ++i) {
             if(ntnMultiplier > 0.0) {
@@ -2501,7 +2501,7 @@ void DataServices::RunTimelinePF(std::map<int, std::string> &mpiFuncNames,
           int bnSize(barrierNames.size());
           bnNumbers.resize(bnSize, 0.0);
           if(bnSize > 0) {
-            bnMultiplier = pow(10, static_cast<int>( 1 + log10(bnSize)));
+            bnMultiplier = std::pow(10, static_cast<int>( 1 + log10(bnSize)));
           }
           for(int i(0); i < bnSize; ++i) {
             if(bnMultiplier > 0.0) {

